### PR TITLE
[ITS] Fix cluster reusage in pile-up events

### PR DESCRIPTION
**Pull request by mpuccio**
Quick fix on the efficiency higher than 1.
Fix for efficiency lower than expected comes in a separate request...

### CODE REVIEW

File: Detectors/ITSMFT/ITS/tracking/include/ITStracking/PrimaryVertexContext.h [hunk 1]
```diff
@@ -27,6 +27,8 @@
 #include "ITStracking/Road.h"
 #include "ITStracking/Tracklet.h"
 
+#include "SimulationDataFormat/MCCompLabel.h"
```
Comment (davidrohr) [inline Detectors/ITSMFT/ITS/tracking/include/ITStracking/PrimaryVertexContext.h]: This breaks CUDA compilation, by pulling in MCCompLabel to GPU compiled file.

Comment (mpuccio) [reply Detectors/ITSMFT/ITS/tracking/include/ITStracking/PrimaryVertexContext.h]: uff, sorry!

Comment (davidrohr) [reply Detectors/ITSMFT/ITS/tracking/include/ITStracking/PrimaryVertexContext.h]: In the long run, I want to make the MC container GPU compatible, but we have to do something about the gsl::span. Should discuss this when @sawenzel is back.

Comment (mpuccio) [reply Detectors/ITSMFT/ITS/tracking/include/ITStracking/PrimaryVertexContext.h]: No issue

**Comment by mpuccio** [comment]
Hi @davidrohr , can we merge this?

**Comment by davidrohr** [comment]
Sorry, didn't see that you updated it. Just give me few minutes to check again, then we can merge.

**Pull request merged**